x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.test.tsx

```diff
@@ -60,14 +60,16 @@ describe('Result', () => {
     ]);
   });
 
-  it('passes through showScore and resultMeta to ResultHeader', () => {
-    const wrapper = shallow(<Result {...props} showScore={true} />);
-    expect(wrapper.find(ResultHeader).prop('showScore')).toBe(true);
-    expect(wrapper.find(ResultHeader).prop('resultMeta')).toEqual({
-      id: '1',
-      scopedId: '1',
-      score: 100,
-      engine: 'my-engine',
+  it('passes showScore, resultMeta, and isMetaEngine to ResultHeader', () => {
```

Comment:
[not a change request since this is how we're already doing it for multiple props] In a vacuum I would think it would be easier to simply access `const { isMetaEngine} = useValues(EngineLogic)` directly from ResultHeader itself instead of passing it down multiple times (ResultView -> Result -> ResultHeader). But I think from this PR I see that using props only lets us make better use of the Library component. That's a good enough reason for me!